config/packages/doctrine.yaml

```diff
@@ -9,13 +9,27 @@ parameters:
 
 doctrine:
     dbal:
-        # configure these for your database server
-        driver: '%database_driver%'
-        charset: '%database_charset%'
-        default_table_options:
-            charset: '%database_charset%'
-            collate: '%database_collation%'
-        url: '%env(resolve:DATABASE_URL)%'
+        connections:
+            default:
+                # configure these for your database server
+                driver: '%database_driver%'
+                charset: '%database_charset%'
+                default_table_options:
+                    charset: '%database_charset%'
+                    collate: '%database_collation%'
+                url: '%env(resolve:DATABASE_URL)%'
+
+            # This connection should be used only by SiteFactory bundle
+            site_factory:
+                # configure these for your database server
+                driver: '%database_driver%'
+                charset: '%database_charset%'
+                default_table_options:
+                    charset: '%database_charset%'
+                    collate: '%database_collation%'
+                url: '%env(resolve:DATABASE_URL)%'
```

Comment:
not following this distinction, it would make sense if we used here different parameters.